pkg/mixer/lint.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2018 mixtool authors
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
+// you may not use this file from except in compliance with the License.
```

Comment:
Seems like a copy-paste-mistake?

Reply:
Genuinely no clue where that's snuck in from, but good catch!